tests/installation/grub_test.pm

```diff
@@ -80,9 +73,21 @@ sub run() {
         }
         send_key 'ret';
     }
-    if (get_var("XEN")) {
+    elsif (get_var("XEN")) {
         send_key_until_needlematch("bootmenu-xen-kernel", 'down', 10, 5);
     }
+    elsif (check_var('ARCH', 'x86_64') && check_var('DISTRI', 'sle')) {
+        record_soft_failure "bsc#995310 - Gnome fails to start";
```

Comment:
this is a very misleading soft failure, because we didn't even tried to start it here